actor/echo/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "echo"
+version = "0.1.0"
+authors = [ "wasmcloud Team" ]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+async-trait = "0.1"
+console_log = "0.2"
+futures = "0.3"
```

Comment:
I know there's nothing we can do about this now, but it still feels like fingernails on a chalkboard to see a single-threaded webassembly module require the use of the `futures` crate. :(